sendsafely/SendSafely.py

```diff
@@ -124,9 +124,14 @@ def get_package_keycode(self, package_id, public_key_id=None, private_key=None):
         url = self.BASE_URL + endpoint
         headers = make_headers(self.API_SECRET, self.API_KEY, endpoint)
         try:
-            keycode = requests.get(url, headers=headers).json()["message"]
+            keycode_json = requests.get(url, headers=headers).json();
+            keycode = keycode_json['message'];
+            keycode_response = keycode_json['response'];
+            if keycode_response == 'FAIL':
+                raise GetKeycodeFailedException(details=str(keycode))
+
             key_pair = pgpy.PGPKey.from_blob(str(private_key))[0]
-            keycode_message = PGPMessage.from_blob(keycode)
+            keycode_message = pgpy.PGPMessage.from_blob(keycode)
```

Comment:
@nwhsendsafely Was this broken prior?

Reply:
@michaeljnowak no, I just added it to align with the other calls. I have reverted and confirmed it still works without the pgpy preappended.